src/view/writer.js

```diff
@@ -887,6 +887,8 @@ function wrapChildren( parent, startOffset, endOffset, attribute ) {
 	let offsetChange = 0;
 
 	for ( let position of wrapPositions ) {
+		position.offset -= offsetChange;
```

Comment:
But I don't know why this is like this. I just recovered a line of code that was mistakenly removed and added a test. What am I going to comment? `// Please do not remove it by mistake again.` ?